plugins/org.eclipse.n4js/src/org/eclipse/n4js/scoping/members/AbstractMemberScope.java

```diff
@@ -188,9 +188,18 @@ protected IEObjectDescription getSingleLocalElementByName(QualifiedName name) {
 			existingMember = findMember(nameAsString, !accessForWriteOperation, !staticAccess);
 		}
 		if (existingMember != null) {
-			return new WrongStaticAccessDescription(
-					EObjectDescription.create(existingMember.getName(), existingMember),
-					staticAccess);
+			boolean wrongStaticAccessDescription = true;
+
+			if (this instanceof MemberScope) {
+				MemberScope thiz = (MemberScope) this;
+				wrongStaticAccessDescription = !thiz.type.isDynamizable();
```

Comment:
`Type#isDynamizable()` will always return "true" for `TN4Classifier`s. You need `TypeRef#isDynamic()`, here.

In the following example you can see the problem:
```
class Foo3 {
//	s: string;
	static s: int;
	m() {}
}
const f3:Foo3/* no plus! */ = new Foo3();
// XPECT FIXME errors --> "The static member s cannot be accessed from a non-static context." at "s"
f3.s; // currently showing: "Couldn't resolve reference to IdentifiableElement 's'."
```

Reply:
Thanks, I changed it!